### DISCUSSION TITLE
Social auth

### DISCUSSION THREAD

**Discussion by Prathik-Jain** [payloadcms/payload#890] [General]
Do we have any docs/blogs/videos which shows us how to integrate social platforms like google and fb for auth on the CMS?

**Comment by jmikrut** [comment]
Hey @Prathik-Jain — not yet! But we would welcome some social auth plugins!

Jump in the Discord if you're up for helping here. That would be great!

**Reply by IRediTOTO** [reply]
Hmm social login is something must have now, why don't build it in core? I must create it by myself again?
